packages/react-components/src/Concepts/Accessibility/AccessibleExperiences.stories.mdx

```diff
@@ -0,0 +1,20 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Concepts/Developer/Accessibility/Experiences" />
+
+## Creating an accessible web application or web page
+
+When designing an application or a web page, keep in mind that the easiest way to make it accessible is to maintain a consistent and standard compliant document structure.
+Following steps help to design an accessible user experience:
+
+- decompose UI to parts and **identify components, variants and behaviors to use**
```

Comment:
This list starts with all lower case characters, while previously they started with upper case, let's follow consistent sentence casing

```suggestion
- Decompose UI to parts and **identify components, variants and behaviors to use**
```